packages/palette/src/elements/StaticCountdownTimer/StaticCountdownTimer.tsx

```diff
@@ -23,13 +21,27 @@ export const StaticCountdownTimer: React.SFC<{
   countdownEnd: string
   currentTime: string
 }> = ({ action, note, countdownEnd, countdownStart, currentTime }) => {
-  const actionDeadline = moment(countdownEnd)
-    .tz(moment.tz.guess())
-    .format("MMM DD, h:mm A z")
+  const dateTime = DateTime.fromISO(countdownEnd).toLocal()
+  const minutes = dateTime.minute < 10 ? "0" + dateTime.minute : dateTime.minute
+  const amPm = dateTime.hour >= 12 ? "pm" : "am"
+  let hour
+  if (dateTime.hour > 12) {
+    hour = dateTime.hour - 12
+  } else if (dateTime.hour === 0) {
+    hour = 12
+  } else {
+    hour = dateTime.hour
+  }
```

Comment:
You could replace this by utilising modulus and a fallback:

```js
> 0 % 12 || 12
12
> 2 % 12 || 12
2
> 12 % 12 || 12
12
> 13 % 12 || 12
1
> 23 % 12 || 12
11
> 24 % 12 || 12
12
```

Reply:
@kierangillen bump

Reply:
@alloy interested to know how you see this as an improvement? Less lines? I prefer my version because I find it more declarative and therefore easier to read the logic.

Reply:
I guess my assumption here is that devs are generally (perhaps even should?) be familiar with concise syntax such as modulus, in which case it is indeed shorter.

I agree that your version spells it all out, though. I’ll leave it up to you, I don’t feel very strong about it, just thought you may have missed the above comment.